### PR TITLE
libminc 2.3 (new formula)

### PR DESCRIPTION
**Pull request by Moisan**
As explained in [issue 2927](https://github.com/Homebrew/homebrew-science/issues/2927), [`minc`](https://github.com/BIC-MNI/minc) has been splitted in two projects: `libminc` and `minc-tools`. This is a new formula for `libminc`.

### CODE REVIEW

**Comment by Moisan** [comment]
`audit` fails due to two cmake files copied by `make install` into the `lib` directory. What would be the correct approach to fix that?

**Comment by dpo** [comment]
If there's no way to prevent `make install` from doing such a thing, the best might be to have libs installed to `libexec/"lib"` and then create symbolic links in `lib`. You can see an example of that in [`asl.rb`, line 41](https://github.com/Homebrew/homebrew-science/blob/master/asl.rb#L41).

**Comment by Moisan** [comment]
@BrewTestBot test this please

**Comment by Moisan** [comment]
@BrewTestBot test this please

**Comment by Moisan** [comment]
Travis fails on API rate limit.
> Error: GitHub API rate limit exceeded for 208.78.110.199.

File: libminc.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    ln_sf Dir["#{libexec}/lib/*.dylib"], lib
+
+    include.mkdir
+    ln_sf Dir["#{libexec}/include/*"], include
+
+    cd "testdir" do
+      system "cmake", "."
+    end
+
+    pkgshare.install %w[testdir]
```
Comment (dpo) [inline libminc.rb]: Just `"testdir"`.

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by dpo** [comment]
```
==> Bottling libminc-2.3.00.yosemite.bottle.tar.gz...
Error: No such file or directory - /usr/local/Cellar/libminc/2.3.00/libexec/include/volume_io/volume_io
```
There's a self reference:
```
$ ls -l /usr/local/Cellar/libminc/2.3.00/libexec/include/volume_io/volume_io
lrwxr-xr-x  1 dpo  admin    28B Jan 17 18:28 /usr/local/Cellar/libminc/2.3.00/libexec/include/volume_io/volume_io -> ../libexec/include/volume_io

```

File: libminc.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  def install
+    system "cmake", ".", "-DCMAKE_INSTALL_PREFIX:PATH=#{libexec}"
+    system "make", "install"
+
+    lib.mkdir
+    ln_sf Dir["#{libexec}/lib/*.a"], lib
+    ln_sf Dir["#{libexec}/lib/*.dylib"], lib
+
+    include.mkdir
+    ln_sf Dir["#{libexec}/include/**/*"], include
```
Comment (dpo) [inline libminc.rb]: It's sufficient to say
```
ln_sf Dir["#{libexec}/include/*"], include
```
but that doesn't resolve the circular reference.

**Comment by Moisan** [comment]
> Error: GitHub API rate limit exceeded

Otherwise everything seems fine.

**Comment by dpo** [comment]
Thanks. Please squash your commits.

**Comment by Moisan** [comment]
@BrewTestBot test this please

**Comment by Moisan** [comment]
@BrewTestBot test this please

**Comment by Moisan** [comment]
@BrewTestBot test this please

**Comment by dpo** [comment]
@BrewTestBot test this please

**Pull request closed without merging**